tensorrt_llm/bench/build/tuning.py

```diff
@@ -50,12 +50,21 @@ def calc_engine_setting(
         Tuple[int, int]: Tuple containing engine configuration information for
         engine build (max_num_tokens, max_batch_size).
     """
+    is_mamba_attn_hybrid = isinstance(model_config, NemotronHybridConfig)
+    if is_mamba_attn_hybrid:
+        # Each mamba cache entry is pretty large (~50MB for 8B model), so we are more conservative when estimating the max batch size
+        kv_cache_gpu_mem_fraction *= kv_cache_gpu_mem_fraction
```

Comment:
Might it be worth moving this into the model configuration also? It could have a generalized method for computing the KV cache scaling based on the model?

Reply:
And by into I mean maybe a method -- not an actual assignment.

Reply:
Good idea. Fixed